projects/shared/src/lib/pipe/join.pipe.ts

```diff
@@ -15,6 +15,26 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-.holding-detail {
-  margin-left: 1em;
+import { Pipe, PipeTransform } from '@angular/core';
+import { isArray } from 'util';
+
+@Pipe({
+  name: 'join',
+  pure: false
+})
+export class JoinPipe implements PipeTransform {
+
+  /**
+   * Transform a string or array to a string with séparator
+   * @param value - string or array
+   * @param separator - string separator
+   * @return string
+   */
+  transform(value: any, separator: string = ' '): any {
+    if (!isArray(value)) {
+      return value;
+    }
+
+    return value.join(separator);
+  }
```

Comment:
Why using a pipe ? It seems a little bit overkill no ? 
Why not just use `{{ arrayData.join(', ') }}`

Reply:
I don't use javascript function into the template.